apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/PartDEventTransformerV2.java

```diff
@@ -44,7 +48,7 @@ static ExplanationOfBenefit transform(
       throw new BadCodeMonkeyException();
     }
 
-    ExplanationOfBenefit eob = transformClaim((PartDEvent) claim);
+    ExplanationOfBenefit eob = transformClaim((PartDEvent) claim, drugCodeDisplayLookup);
```

Comment:
Probably need to do some Javadoc (in multiple files); at least certainly the methods that you've added the drug display code to.